src/sql/workbench/contrib/notebook/browser/notebookViews/notebookViews.contribution.ts

```diff
@@ -0,0 +1,26 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { Registry } from 'vs/platform/registry/common/platform';
+import { localize } from 'vs/nls';
+import { IConfigurationRegistry, Extensions as ConfigExtensions } from 'vs/platform/configuration/common/configurationRegistry';
+
+export const NOTEBOOK_VIEWS_ENABLED_PROPERTY = 'notebookViews.enabled';
+
+const properties = {};
+properties[NOTEBOOK_VIEWS_ENABLED_PROPERTY] = {
+	'type': 'boolean',
+	'default': false,
+	'description': localize('notebookViews.enabled', "(Preview) enable notebook views")
```

Comment:
Given that this isn't actually being used yet I'd suggest not adding it until that functionality is added so as to not cause confusion about what it does